common/web/gesture-recognizer/src/test/auto/browser/cases/recordedCoordSequences.js

```diff
@@ -39,6 +44,23 @@ describe("Layer one - DOM -> InputSequence", function() {
       let resultPromise;
 
       let playbackEngine = new InputSequenceSimulator(this.controller);
+
+      // **********************************
+      // Android-Chrome sequence simulation does not allow fractional values in MouseEvent clientX/clientY...
```

Comment:
I think real mouse events are always pixel aligned on all platforms, aren't they? So I'm not really understanding why this is so big an issue?

Reply:
The affected automated tests check, among other things, how boundary conditions are handled and respected over the lifetime of the touchpath.  If the reproduced path's alignment is off, it's possible for a sequence that was in-bounds originally to be misaligned enough to go partly out of bounds... or even just out of bounds early.